src/graphql/typeDefs/index.ts

```diff
@@ -0,0 +1,11 @@
+import { readFile } from 'fs-extra';
```

Comment:
I was in the middle of replacing this with `graphql-import`'s `importSchema` tool, so this is going to be deleted soon.